.babelrc

```diff
@@ -15,27 +15,46 @@
          */
         "browsers": ["Chrome 61", "Firefox 60", "iOS 11", "Edge 16", "ChromeAndroid 67", "Safari 11"]
```

Comment:
not a big deal but this doesn't match the comment above

Reply:
You mean the two browsers arrays? They're intentionally different.

Reply:
I see now thanks .